src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -108,4 +173,13 @@ public void exec(final URI home) throws IOException {
             }
         );
     }
+
+    /**
+     * Create a joiner for a header.
+     * @return Joiner
+     */
+    private Joiner joiner() {
```

Comment:
@lautarobock instead of extracting this into a method, you should create a private static final String with the `\r\n` string.